templates/zerver/help/enable-or-disable-pressing-enter-to-send.md

```diff
@@ -1,21 +1,20 @@
 # Press Enter to send
 
-By default, Zulip sends your message when you press enter (or return)
-on your keyboard. If you want to turn this off, simply uncheck the
-**Press Enter to send** box below the compose field:
+By default, Zulip sends your message when you press the **Send** button in
+the compose box. If you would like to send your message by pressing
```

Comment:
You should just make another section titled `## Enable  pressing enter to send` and begin the section with "If you would like to send your message..."